RedHat-UX/red-hat-design-system
# docs: added usage code to blockquote

## What I did

1. Added a usage code sample to the code tab on blockquote's code tab


## Testing Instructions

1. Go to https://deploy-preview-1160--red-hat-design-system.netlify.app/elements/blockquote/code/#usage
2. Verify that there is a code block showing the rh-blockquote example.
